docs-src/python/interrupt-a-workflow-execution.md

```diff
@@ -0,0 +1,49 @@
+---
+id: interrupt-a-workflow-execution
+title: How to interrupt a Workflow Execution
+description:
+sidebar_label: Interrupt a Workflow Execution
+tags:
+  - guide-context
+---
+
+You can interrupt a Workflow Execution in one of the following ways:
```

Comment:
AFAIK we have not used this "interrupt" terminology before. I'm not necessarily against introducing it, but, it might be worth discussing more broadly.

Reply:
I think just cancel and terminate should be documented alongside other things you can do with a workflow, I don't think we need to group them together with some term that describes both.

Also, are you planning on putting this mostly-non-python-specific docs in all languages or is this non-language-specific description of cancel and terminate in just this language-specific guide?